README.md

```diff
@@ -100,31 +98,18 @@ qiime chemistree predict-fingerprints --p-sirius-path 'sirius-osx64-4.0.1/bin' \
 
 ```bash
 qiime chemistree make-hierarchy \
-  --i-collated-fingerprints collated_fingerprints_qc.qza \
-  --p-prob-threshold 0.5 \
-  --o-tree demo-chemisTree.qza
-```
-
-This generates a tree relating the MS1 features in these data based on molecular substructures predicted for MS1 features. This is of type `Phylogeny[Rooted]`. **Note**: SIRIUS predicts molecular substructures for a subset of features (typically for 70-90% of all MS1 features) in your experiment (based on factors such as sample type, the quality MS2 spectra, and used-defined tolerances such as `--p-ppm-max`, `--p-zodiac-threshold`). Thus, we need to remove the MS1 features without fingerprints from the feature table with:
-
-```bash
-qiime chemistree match-table --i-tree demo-chemisTree.qza \
+  --i-csi-result fingerprints.qza \
   --i-feature-table feature-table.qza \
-  --o-filtered-feature-table filtered-feature-table.qza
+  --o-tree demo-chemisTree.qza \
+  --o-matched-feature-table filtered-feature-table.qza
 ```
 
-This filters the MS1 table to include only the MS1 features with molecular fingerprints. The resulting table is also of type `FeatureTable[Frequency]`.
+This method performs two tasks:
+1. Generates a tree relating the MS1 features in these data based on molecular substructures predicted for MS1 features. This is of type `Phylogeny[Rooted]`. By default, we only use PUBCHEM fingerprints (total 489 molecular properties). Adding `--p-no-qc-properties` retains all (2936) the molecular properties in the contingency table.
+2. Filters the MS1 features without fingerprints from the feature table such that the feature IDs in the feature table and the tree match. This is done because SIRIUS predicts molecular substructures for a subset of features (typically for 70-90% of all MS1 features) in an experiment (based on factors such as sample type, the quality MS2 spectra, and user-defined tolerances such as `--p-ppm-max`, `--p-zodiac-threshold`). The resulting feature table is also of type `FeatureTable[Frequency]`.
 
-Thus, using these steps, we can generate a tree (`demo-chemisTree.qza`) relating MS1 features in mass-spectrometry dataset along with a matched feature table (`filtered-feature-table.qza`). These can be used as inputs to perform [UniFrac](https://aem.asm.org/content/71/12/8228)-based [alpha-diversity](https://docs.qiime2.org/2018.8/plugins/available/diversity/alpha-phylogenetic/) and [beta-diversity](https://docs.qiime2.org/2018.8/plugins/available/diversity/beta-phylogenetic/) analyses.
+Thus, using these steps, we can generate a tree (`demo-chemisTree.qza`) relating MS1 features in a mass-spectrometry dataset along with a matched feature table (`filtered-feature-table.qza`). These can be used as inputs to perform chemical phylogeny-based [alpha-diversity](https://docs.qiime2.org/2018.8/plugins/available/diversity/alpha-phylogenetic/) and [beta-diversity](https://docs.qiime2.org/2018.8/plugins/available/diversity/beta-phylogenetic/) analyses.

```

Comment:
This link to qiime2 2018.8, is that accurate?

Reply:
Thanks for noting this - will replace these links to something general as this will keep changing

Reply:
.. or link to the version of qiime2 used for the demo